src/subarray.jl

```diff
@@ -6,12 +6,18 @@ isordered(sa::SubArray{T,N,P}) where {T,N,P<:CategoricalArray} = isordered(paren
 levels!(sa::SubArray{T,N,P}, newlevels::Vector) where {T,N,P<:CategoricalArray} =
     levels!(parent(sa), levels)
 
+if VERSION ≥ v"0.7.0-"
```

Comment:
That's `0.7.0-DEV.3020`. Also, given that Femtocleaner will remove 0.6-only blocks for us in a few weeks, I'd rather duplicate the full `unique` body instead of defining `_subarray_indices`, which Femtocleaner won't remove automatically for us.

Reply:
ok, will change.